qemu/tests/x86_cpu_flags.py

```diff
@@ -23,11 +23,15 @@ def run(test, params, env):
         check_cpu_flags(params, flags, test)
 
     unsupported_models = params.get("unsupported_models", "")
+    check_models = params.get("check_models", "")
+    skip_models = params.get("skip_models", "")
     cpu_model = params.get("cpu_model")
     if not cpu_model:
         cpu_model = cpu.get_qemu_best_cpu_model(params)
     if cpu_model in unsupported_models.split():
         test.cancel("'%s' doesn't support this test case" % cpu_model)
+    if skip_models.split() and cpu_model in skip_models.split():
+        params["cpu_model"] = check_models.split()[0]
```

Comment:
IMO, test can be canceled if cpu_model is in skip_models.

Reply:
> IMO, test can be canceled if cpu_model is in skip_models.

No, we can't, if we do so we will skip this test on Milan host. But we can use the old model on Milan host to test this.

Reply:
For the code design,  the skip_models is not for EPYC_Milan only,  if you want to fall back to the old model,  then coupled cpu models should be specified.